keps/sig-network/2595-expanded-dns-config/README.md

```diff
@@ -152,14 +152,17 @@ Verified that the API server accepts the pod or podTemplate with the expanded
 DNS config and the kubelet accepts the resolv.conf or pod with the expanded DNS
 config.
 
+- `k8s.io/kubernetes/pkg/kubelet/network/dns/dns.go`: `2023/04/22` - `68.7%`
+
 ##### Integration tests
 
 No integration tests are planned.
 
 ##### e2e tests
 
-Will add an e2e test to ensure that the pod with the expanded DNS config can be
-created and run successfully.
+Tests in `test/e2e/network/dns.go`:
+
+- [It] [sig-network] DNS should work with the pod containing more than 6 DNS search paths and longer than 256 search list characters: [results](https://storage.googleapis.com/k8s-triage/index.html?test=should%20work%20with%20the%20pod%20containing%20more%20than%206%20DNS%20search%20paths%20and%20longer%20than%20256%20search%20list%20characters)
```

Comment:
the test flakes sometimes. Are those flakes known? Can those be eliminated for GA?

Reply:
the failures are associated to the jobs `e2e-kops-grid-gce-calico-u2004-k26	` that has another 30 tests failing too

Reply:
> the list should contain only stable jobs, the ones in the release blocking and informing dashboard

I don't think non-GA feature may be in release informing dashboard. So if this is a requirement, it creates a chicken-and-egg system.

Also btw, the question for GA promotion is always whether there is any tests that needs to be made into Conformance. I assumed no, but maybe need to be said explicitly

Reply:
Actually, the test is already a part of the `pull-kubernetes-e2e-kind`, `pull-kubernetes-e2e-kind-ipv6`, `pull-kubernetes-e2e-gce`, and so on which are the pre-submit tests.

I think the dashboard only captures the failure cases. (The name of the dashboard is `Kubernetes Aggregated Failures`)

Reply:
> the test flakes sometimes. Are those flakes known? Can those be eliminated for GA?

I think we don't need to eliminate all of the test failures as far as the failing job is not critical.

Reply:
Which jobs are not critical? Then they and those flaky tests will get removed because they don't work reliably? That doesn't sound right.

Reply:
> Which jobs are not critical?

There are some jobs that are flaky even without running this test. I think that they are not critical in terms of this KEP.
- e2e-kops-grid-gce-calico-u2004-k26
- e2e-kops-grid-cilium-eni-amzn2-k26

Also, it seems that the related repositories are not part of the kubernetes org. I think we can ignore them... right?

Reply:
to be clear, a test is flaky if is the test itself flaky, those job have multiple failures and the tooling does not differentiate if the test is flaky or the job is.
If there is a job that fails a lot of tests constntly, this will increase the stats of an individual test, that is wrong but we do not do anything for filtering those out.
Flaky jobs should be removed or suppresed from the reporting tools

Reply:
thanks for the clarification.

fixed it to only contain ci-kubernetes-kind-e2e-parallel which is in the release blocking dashboard.